sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -1,7 +1,13 @@
 # Release History
 
-## 4.1.0-preview.2 (Unreleased)
```

Comment:
We should not be changing the changelog that was put in place for past releases.
Add a new section for the GA version and note the net changes compared to the previous GA version

Reply:
My mistake. I didnt realize that 4.1.0-preview.2 was not released. Therefore, it makes perfect sense that we change this header to from `4.1.0-preview.2` to `4.1.0`